LICENSE

```diff
@@ -175,3 +175,98 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```

Comment:
+1 seems to match https://github.com/apache/incubator-gobblin/blob/master/LICENSE#L179